tensorflow/python/ops/clip_ops.py

```diff
@@ -194,7 +194,9 @@ def clip_by_norm(t, clip_norm, axes=None, name=None):
   Args:
     t: A `Tensor` or `IndexedSlices`.  This must be a floating point type.
     clip_norm: A 0-D (scalar) `Tensor` > 0. A maximum clipping value, also
-      floating point
+      floating point.
+      Note: If clip_norm given as a negative value then it will be converted
```

Comment:
- clip_norm given -> clip_norm is given
- Zero -> zero
- internally and exception will not be raised here -> without raising an exception

Alternatively, just say:
> If a negative clip_norm is provided, it will be treated as zero.